content/api-reference/api-console.md

```diff
@@ -4,61 +4,115 @@ uid: apiConsole
 
 # API console
 
-The API Console provides a graphical interface for using the REST API. When using the API Console, you select a GET, POST, PUT, DELETE, or PATCH action, select the objects for the action, and execute.
+The API Console provides a graphical interface for using the AVEVA Data Hub REST API. Developers can use this console to configure and test API requests before implementing them in their own applications. To use the API Console, select select **Developer Tools** > **API Console** from the left pane.
 
-To use the API Console:
+## Request area
 
-1. In the left pane, select **Developer Tools** > **API Console**.
+Use the request area to configure an API request. Use this space to choose a version of the API, the request scope, the request verb, and the URI endpoint. This area is also used to configure the request header, body, and parameters.
 
-1. Select a version from the dropdown list.
+![request area](../_images/request-area.png)
 
-1. Select the **Root/Tenant path enabled** icon to toggle between a tenant-scoped path or a root-scoped path.
+### API version
 
-1. Select an action from the **GET/POST/PUT/DELETE/PATCH** dropdown list.
+Select a version from the dropdown list. For more information on the available versions, see [AVEVA Data Hub API versioning](xref:osisoftCloudServices#aveva-data-hub-api-versioning).
 
-1. Type a slash (/) in the text field to display a dropdown list.
+### Tenant path
 
-1. If you selected a tenant-scoped path in the second step, you can take any of the following actions:
+Select the **Root/Tenant path enabled** ![root/tenant path enabled](../_icons/default/home.svg) icon to toggle between a tenant-scoped path or a root-scoped path. The **Full Path** field (which is readonly) updates according to your selection.
```

Comment:
Is the difference between tenant-scope and root-scoped considered common knowledge or should you define that?

Reply:
I'm going to need to ask around a bit to find out the difference.